lib/syntax/node/Block.js

```diff
@@ -60,8 +75,7 @@ export const structure = {
     ]]
 };
 
-export function parse(isStyleBlock) {
-    const consumer = isStyleBlock ? consumeDeclaration : consumeRule;
+export function parse(isStyleBlock, { allowNestedRules = false } = {}) {
```

Comment:
I needed to add a new parameter because previously `isStyleBlock` meant it could only contain declarations, which is still true for at-rules but not for regular rules.